docs/Installation/Installation.md

```diff
@@ -149,55 +149,58 @@ To build with the Docker image:
 
 2. Retrieve the Docker image (you may need `sudo` in front of this command)
    ```
-   docker pull sxscollaboration/spectrebuildenv:latest
+   docker pull sxscollaboration/spectre:demo
```

Comment:
We should use the `dev` container here, because the intention is to clone the spectre repo (often on the host file system), edit code, and build (in the container). The demo container doesn't work well for this, because it already has a cloned spectre repo (in the container). So please change `demo` -> `dev` here. You can probably move some or most of the other edits you made in this file to the "Hitchhiker's" guide. I'll review the changes to the guide then.